metadata-ingestion/tests/unit/test_glue_source.py

```diff
@@ -61,10 +66,23 @@ def test_column_type(hive_column_type: str, expected_type: Type) -> None:
     assert type(actual_schema_field_type.type) == expected_type
 
 
+@pytest.mark.parametrize(
+    "platform_instance, mce_file, mce_golden_file",
```

Comment:
This is interesting! What values will be supplied to the first two params, `tmp_path` & `pytestconfig`?

Reply:
To be honest, I haven't looked into them, just assumed they were provided by `pytest` 😅

Looking into this now, I have found that there are some built-in fixtures supporting them: 
`tmp_path` https://docs.pytest.org/en/6.2.x/tmpdir.html
`pytestconfig` https://docs.pytest.org/en/latest/reference/reference.html#pytestconfig